server/tests/auth/test_github_signup.py

```
@@ -0,0 +1,398 @@
+        assert user_id == "new-user"
+        assert "/sign-in?handoff=" in response.location
+        role.assert_called_once_with("new-user", "admin", "new-org")
+        conn.commit.assert_called_once()
+        sql = " ".join(c.args[0] for c in cur.execute.call_args_list)
+        assert "signup_handoff_hash" in sql        # handoff minted
+        assert "github_installations" in sql       # install linked
+        assert "email_verified" in sql             # provisioned verified
+
+    def test_new_user_has_unusable_password(self, signup_app):
+        app, mod = signup_app
+        db_pool, conn, cur = _mock_db(
```

Ruff (0.16.1)

[warning] Unpacked variable `conn` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)